src/vmm/src/lib.rs

```diff
@@ -103,12 +108,18 @@ pub const DEFAULT_KERNEL_CMDLINE: &str = "panic=1 pci=off";
 #[cfg(target_arch = "aarch64")]
 /// Default kernel command line.
 pub const DEFAULT_KERNEL_CMDLINE: &str = "reboot=t panic=1 pci=off";
+///Defualt address allocator alignment. It needs to be a power of 2.
+pub const DEFAULT_ADDRESSS_ALIGNEMNT: u64 = 4;
+///Default allocation policy for address allocator.
+pub const DEFAULT_ALLOC_POLICY: AllocPolicy = AllocPolicy::FirstMatch;
 
 /// VMM memory related errors.
 #[derive(Debug)]
 pub enum MemoryError {
     /// Not enough memory slots.
     NotEnoughMemorySlots,
+    ///AddressAllocatorError
```

Comment:
```suggestion
    /// AddressAllocatorError
```
A space is missing here and in some of the comments below.